README.adoc

```diff
@@ -21,7 +21,24 @@ Check back for updates, we plan to have something available "soon".
 [[documentation]]
 == Documentation
 
-- link:./developing.adoc[Developing CodeReady Containers]
+=== Using CodeReady Containers
+
+You can find the raw documentation files link:./docs/source[here]
```

Comment:
```suggestion
You can find the source files for the documentation in the link:./docs/source[docs/source] directory.
```

Reply:
done

Reply:
I don't see these changes. Other than that, everything looks good. :+1:

Reply:
ah, missed that one. done!